### PR TITLE
Admin application submitted view

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/page-applications.scss [hunk 1]
```diff
@@ -42,3 +42,155 @@
...
+    text-align: left;
+    text-indent: -9999px;
+  }
+
+  .glyphicon-positive {
+    background-image: image-url("icon-grade-positive.png");
+    @include is-retina {
+      background-image: image-url("icon-grade-positive@2.png");
+    }
+  }
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Rule declaration should be followed by an empty line

File: app/assets/stylesheets/admin/page-applications.scss [hunk 2]
```diff
@@ -42,3 +42,155 @@
...
+    height: 20px;
+    margin-top: -10px;
+    background-size: 20px 20px;
+    text-align: left;
+    text-indent: -9999px;
+  }
+
+  .glyphicon-positive {
+    background-image: image-url("icon-grade-positive.png");
+    @include is-retina {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: @include declaration should be preceded by an empty line

File: app/assets/stylesheets/admin/page-applications.scss [hunk 3]
```diff
@@ -42,3 +42,155 @@
...
+      background-image: image-url("icon-grade-positive@2.png");
+    }
+  }
+  .glyphicon-average {
+    background-image: image-url("icon-grade-average.png");
+    @include is-retina {
+      background-image: image-url("icon-grade-average@2.png");
+    }
+  }
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Rule declaration should be followed by an empty line

File: app/assets/stylesheets/admin/page-applications.scss [hunk 4]
```diff
@@ -42,3 +42,155 @@
...
+
+  .glyphicon-positive {
+    background-image: image-url("icon-grade-positive.png");
+    @include is-retina {
+      background-image: image-url("icon-grade-positive@2.png");
+    }
+  }
+  .glyphicon-average {
+    background-image: image-url("icon-grade-average.png");
+    @include is-retina {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: @include declaration should be preceded by an empty line

File: app/assets/stylesheets/admin/page-applications.scss [hunk 5]
```diff
@@ -42,3 +42,155 @@
...
+      background-image: image-url("icon-grade-average@2.png");
+    }
+  }
+  .glyphicon-negative {
+    background-image: image-url("icon-grade-negative.png");
+    @include is-retina {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: @include declaration should be preceded by an empty line

File: app/assets/stylesheets/admin/page-applications.scss [hunk 6]
```diff
@@ -42,3 +42,155 @@
...
+        }
+
+        .active-arrow {
+          display: block;
+        }
+      }
+    }
+  }
+
+  #case-grading .form-label {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 7]
```diff
@@ -42,3 +42,155 @@
...
+
+      .form-value,
+      .form-edit-link {
+        display: none;
+      }
+    }
+  }
+}
+
+#financial-summary .form-group {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Avoid using id selectors

File: app/assets/stylesheets/admin/page-applications.scss [hunk 8]
```diff
@@ -42,3 +42,155 @@
...
+    margin-bottom: 20px;
+    border-bottom: none;
+    font-size: 11px;
+
+    > li {
+      > a {
+        margin: 0;
+        margin-left: 5px;
+
+        &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 9]
```diff
@@ -42,3 +42,155 @@
...
+        &:focus {
+          padding: 3px 10px;
+          border: 2px solid #aaa;
+          border-radius: 100px;
+          color: #aaa;
+        }
+
+        &:hover {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 10]
```diff
@@ -42,3 +42,155 @@
...
+          border: 2px solid #666;
+          background: transparent;
+          color: #666;
+        }
+
+        .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5

File: app/assets/stylesheets/admin/page-applications.scss [hunk 11]
```diff
@@ -42,3 +42,155 @@
...
+          background-size: 8px 4px;
+
+          @include is-retina {
+            background-image: image-url("icon-arrow-tab@2.png");
+          }
+        }
+      }
+
+      &.active > a {
+        &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 12]
```diff
@@ -42,3 +42,155 @@
...
+        &:hover,
+        &:focus {
+          border: 2px solid #666;
+          color: #666;
+        }
+
+        .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5

File: app/assets/stylesheets/admin/page-applications.scss [hunk 13]
```diff
@@ -42,3 +42,155 @@
...
+  table.table {
+    margin-bottom: 0;
+  }
+}
+
+.form-edit-link {
+  font-size: 13px;
+
+  .glyphicon {
+    font-size: 10px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Properties should be ordered margin-right, font-size

File: app/assets/stylesheets/admin/page-applications.scss [hunk 14]
```diff
@@ -42,3 +42,155 @@
...
+
+#financial-summary .form-group {
+  margin-bottom: 30px;
+  border-bottom: none;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  table.table {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 15]
```diff
@@ -42,3 +42,155 @@
...
+  margin-top: 20px;
+}
+
+.submitted-view {
+  .nav-tabs {
+    margin-bottom: 20px;
+    border-bottom: none;
+    font-size: 11px;
+
+    > li {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 16]
```diff
@@ -42,3 +42,155 @@
...
+      > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 17]
```diff
@@ -42,3 +42,155 @@
...
+          background-image: image-url("icon-arrow-tab.png");
+          background-size: 8px 4px;
+
+          @include is-retina {
+            background-image: image-url("icon-arrow-tab@2.png");
+          }
+        }
+      }
+
+      &.active > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 18]
```diff
@@ -42,3 +42,155 @@
...
+  }
+  .glyphicon-negative {
+    background-image: image-url("icon-grade-negative.png");
+    @include is-retina {
+      background-image: image-url("icon-grade-negative@2.png");
+    }
+  }
+
+  .js & {
+    .form-group {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 19]
```diff
@@ -42,3 +42,155 @@
...
+      padding-bottom: 10px;
+      border-bottom: 1px solid #999;
+
+      &:last-child {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 20]
```diff
@@ -42,3 +42,155 @@
...
+        margin-bottom: 0;
+        padding-bottom: 0;
+        border-bottom: none;
+      }
+    }
+
+    .form-control {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 21]
```diff
@@ -42,3 +42,158 @@
...
+        background-image: image-url("icon-arrow-tab.png");
+        background-size: 8px 4px;
+
+        @include is-retina {
+          background-image: image-url("icon-arrow-tab@2.png");
+        }
+      }
+    }
+
+    > li.active > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 22]
```diff
@@ -42,3 +42,158 @@
...
+  margin-top: 20px;
+}
+
+.submitted-view {
+  .nav-tabs {
+    margin-bottom: 20px;
+    border-bottom: none;
+    font-size: 11px;
+
+    > li > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 23]
```diff
@@ -42,3 +42,158 @@
...
+      margin: 0;
+      margin-left: 5px;
+
+      &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 24]
```diff
@@ -42,3 +42,158 @@
...
+      &:focus {
+        padding: 3px 10px;
+        border: 2px solid #aaa;
+        border-radius: 100px;
+        color: #aaa;
+      }
+
+      &:hover {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 25]
```diff
@@ -42,3 +42,158 @@
...
+        border: 2px solid #666;
+        background: transparent;
+        color: #666;
+      }
+
+      .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 5

File: app/assets/stylesheets/admin/page-applications.scss [hunk 26]
```diff
@@ -42,3 +42,158 @@
...
+
+    > li.active > a {
+      &,
+      &:hover,
+      &:focus {
+        border: 2px solid #666;
+        color: #666;
+      }
+
+      .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 5

File: app/assets/stylesheets/admin/page-applications.scss [hunk 27]
```diff
@@ -42,3 +42,158 @@
...
+        display: block;
+      }
+    }
+  }
+
+  .section-case-grading .form-label {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 28]
```diff
@@ -42,3 +42,158 @@
...
+        padding-bottom: 0;
+        border-bottom: none;
+      }
+    }
+
+    .form-control {
+      display: none;
+    }
+
+    .form-edit {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 29]
```diff
@@ -42,3 +42,158 @@
...
+      border-bottom: none;
+
+      .form-control {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 30]
```diff
@@ -42,3 +42,158 @@
...
+        display: block;
+      }
+
+      .form-value,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/admin/page-applications.scss [hunk 31]
```diff
@@ -42,3 +42,158 @@
...
+
+#financial-summary .form-group {
+  margin-bottom: 30px;
+  border-bottom: none;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  .table {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 32]
```diff
@@ -42,3 +42,193 @@
...
+      background-size: 8px 4px;
+
+      @include is-retina {
+        background-image: image-url("icon-arrow-tab@2.png");
+      }
+    }
+
+    &.active {
+      > a {
+        &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 33]
```diff
@@ -42,3 +42,193 @@
...
+  margin-bottom: 20px;
+  border-bottom: none;
+  font-size: 11px;
+
+  .no-js & {
+    display: none;
+  }
+
+  > li {
+    > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 34]
```diff
@@ -42,3 +42,193 @@
...
+      margin: 0;
+      margin-left: 5px;
+
+      &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 35]
```diff
@@ -42,3 +42,193 @@
...
+      &:focus {
+        padding: 3px 10px;
+        border: 2px solid #aaa;
+        border-radius: 100px;
+        color: #aaa;
+      }
+
+      &:hover {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 36]
```diff
@@ -42,3 +42,193 @@
...
+        border: 2px solid #666;
+        background: transparent;
+        color: #666;
+      }
+    }
+
+    .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 37]
```diff
@@ -42,3 +42,193 @@
...
+      background-image: image-url("icon-arrow-tab.png");
+      background-size: 8px 4px;
+
+      @include is-retina {
+        background-image: image-url("icon-arrow-tab@2.png");
+      }
+    }
+
+    &.active {
+      > a {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 38]
```diff
@@ -42,3 +42,193 @@
...
+        &,
+        &:hover,
+        &:focus {
+          border: 2px solid #666;
+          color: #666;
+        }
+      }
+
+      .active-arrow {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 39]
```diff
@@ -42,3 +42,193 @@
...
+        display: block;
+      }
+    }
+  }
+}
+
+.no-js .tab-content > .tab-pane {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 40]
```diff
@@ -42,3 +42,193 @@
...
+  display: block;
+  margin-bottom: 30px;
+  visibility: visible;
+
+  &:last-child {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 41]
```diff
@@ -42,3 +42,193 @@
...
+  .glyphicon-negative {
+    background-image: image-url("icon-grade-negative.png");
+
+    @include is-retina {
+      background-image: image-url("icon-grade-negative@2.png");
+    }
+  }
+
+  .no-js & {
+    .form-value,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/admin/page-applications.scss [hunk 42]
```diff
@@ -42,3 +42,193 @@
...
+      background-image: image-url("icon-arrow-tab.png");
+      background-size: 8px 4px;
+
+      @include is-retina {
+        background-image: image-url("icon-arrow-tab@2.png");
+      }
+    }
+  }
+
+  > li.active {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Merge rule `> li.active` with rule on line 61<br>Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/page-applications.scss [hunk 43]
```diff
@@ -42,3 +42,193 @@
...
+    > a {
+      &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Selector should have depth of applicability no greater than 2, but was 3

**Comment by RuslanHamidullin** [comment]
@hammerfunk looks fine for me :+1:

**Pull request merged**